zephyr/module.yml

```diff
@@ -1,4 +1,9 @@
+# Copyright (c) 2022 Legrand North America, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+
 samples:
   - boot/zephyr
 build:
+  kconfig: zephyr/Kconfig
```

Comment:
Why can't refer to `../boot/zephyr/Kconfig` ?
Was this move neccessery?

Reply:
In the original implementation the boot/zephyr/Kconfig file mixed two layers into one file.  First, the top-level (executable-specific) build Kconfig declarations, which is optional and usually does little more than sources "Kconfig.zephyr".  Second, the module-level Kconfig declarations, which declare controls for configuring subsystems. Combining them caused problems when configuring and building my MCUBoot executable from outside of the MCUBoot repository.